example/android/gradle.properties

```diff
@@ -1,3 +0,0 @@
-org.gradle.jvmargs=-Xmx1536M
-android.useAndroidX=true
-android.enableJetifier=true
```

Comment:
Why is this removed? The gradle is complaining about not using androidX. This can create build issues.